addons/mail/static/src/messaging/component/notification_popover/notification_popover.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.messaging.component.NotificationPopover" owl="1">
+        <div class="o_NotificationPopover">
+            <t t-foreach="notifications" t-as="notification" t-key="notification.localId">
+                <div>
+                    <span name="notification_status" class="o_NotificationPopover_icon">
```

Comment:
this looks like `iconContainer`, `icon` should be on `<i>`, and `name` should be the same as the part in BEM class, so `iconContainer` too

Reply:
I'll refactor this whole template, and create a new one for snailmail instead of overriding the original one to completely replace it, which didn't make a lot of sense.